tests/test_umbral.py

```diff
@@ -18,15 +19,43 @@ def test_decapsulation_by_alice():
     priv_key = pre.gen_priv()
     pub_key = pre.priv2pub(priv_key)
 
-    sym_key, capsule = pre.encapsulate(pub_key)
+    sym_key, capsule = pre._encapsulate(pub_key)
     assert len(sym_key) == 32
 
     # The symmetric key sym_key is perhaps used for block cipher here in a real-world scenario.
 
-    sym_key_2 = pre.decapsulate_original(priv_key, capsule)
+    sym_key_2 = pre._decapsulate_original(priv_key, capsule)
     assert sym_key_2 == sym_key
 
 
+@pytest.mark.parametrize("N,threshold", parameters)
+def test_simple_api(N, threshold):
+    params = umbral.UmbralParameters()
+    pre = umbral.PRE(params)
```

Comment:
Just getting going on a more thorough review.  Overall, this is a *huge* improvement from 24 hours ago!

I'll make Issues for things that are, at least to me, outstanding.  I started on this line, with #31.